app/models/user.rb

```diff
@@ -1,10 +1,18 @@
 class User < ApplicationRecord
-  before_save{email.downcase!}
+  before_save{self.email = email.downcase}
   validates :name, presence: true, length: {maximum: 50}
```

Comment:
`50` `255` ..
các giá trị này, em phải khai báo vào file Settings chứ

nếu chưa có file settings thì tìm hiểu `gem config in rails` trước  nhé